resources/velero/templates/restic-daemonset.yaml

```diff
@@ -0,0 +1,98 @@
+{{- if .Values.deployRestic }}
```

Comment:
why the file is called "restic-***"?

Reply:
Oh, it was heavily based on https://github.com/helm/charts/blob/master/stable/velero/templates/restic-daemonset.yaml. I changed it to `daemonset.yaml`